templates/kubernetes/managed.html

```diff
@@ -59,7 +59,7 @@ <h3>Resources</h3>
               <img class="p-heading-icon__img p-heading-icon__img--small" src="{{ ASSET_SERVER_URL }}75902002-news-feed-strip-stock_ebook.svg" />
               <h4 class="p-heading-icon__title">Datasheet</h4>
             </div>
-            <h3 class="p-heading--four"><a class="p-link--external" href="{{ ASSET_SERVER_URL }}b35eca50-Enterprise-Kubernetes-datasheet.pdf" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'External Link', 'eventAction' : 'Managed cloud', 'eventLabel' : 'Kubernetes for the enterprise', 'eventValue' : undefined });">Kubernetes for the enterprise</a></h3>
+            <h3 class="p-heading--four"><a class="p-link--external" href="{{ ASSET_SERVER_URL }}223eab1a-canonical-kubernetes-enterprise-2018-25-07.pdf" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'External Link', 'eventAction' : 'Managed cloud', 'eventLabel' : 'Kubernetes for the enterprise', 'eventValue' : undefined });">Kubernetes for the enterprise</a></h3>
```

Comment:
Please drop the existing `onclick` event as they are now automatically generated on all links, with a new naming convention.